traitsui/tests/editors/test_html_editor.py

```diff
@@ -74,32 +255,129 @@ def test_base_url_changed(self):
             model.model_base_url = "/new_dir"
 
     @requires_toolkit([ToolkitName.qt])
-    @mock.patch('webbrowser.open_new')
-    def test_open_externally_qt(self, open_new_mock):
-        from pyface.qt import QtCore, QtWebKit
+    def test_open_internal_link(self):
+        # this test requires Qt because it relies on the link filling up
+        # the entire page through the use of CSS, which isn't supported
+        # by Wx.
+        model = HTMLModel(content="""
+        <html>
+            <a
+              href='/#'
```

Comment:
~why is `href='/#' here, but in the `test_open_internal_link_externally` test it is `href='test://testing'`?~

EDIT: I think I understand now